server.js

```diff
@@ -1,23 +1,187 @@
 const express = require("express");
 const cors = require("cors");
+//const { check, validationResult } = require("express-validator");
 
 const app = express();
-
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
 app.use(cors());
 
 const welcomeMessage = {
   id: 0,
   from: "Bart",
-  text: "Welcome to CYF chat system!",
+  text: "Welcome to CYF chat system!"
 };
 
 //This array is our "data store".
 //We will start with one message in the array.
 //Note: messages will be lost when Glitch restarts our server.
-const messages = [welcomeMessage];
+const messages = [  welcomeMessage,
+  {
+    id: 1,
+    from: "Joanna",
+    text: "Hey, how are you?"
+  },
+  {
+    id: 2,
+    from: "Tas",
+    text: "Are you going us today"
+  },
+  
+  {
+    id: 3,
+    from: "Altom",
+    text: "Are you going us today"
+  }
+  
+];
 
-app.get("/", function (request, response) {
+app.get("/", function(request, response) {
   response.sendFile(__dirname + "/index.html");
 });
 
-app.listen(process.env.PORT);
+// const randomId = (){
+//   return Math.floor(Math.random() * 1000000000);
+// }
+
+//read  all messages
+
+app.get("/messages", (req, res) => {
+  res.json(messages);
+});
+
+// create message
+app.post("/messages", (req, res) => {
+  let today = new Date();
```

Comment:
in general, unless you need to reassign a variable it is preferable to use `const` rather than `let` - this makes it clear that the variable always points to the same value or object